projects/igniteui-angular/src/lib/grids/toolbar/grid-toolbar-hiding.component.html

```diff
@@ -0,0 +1,13 @@
+<ng-container *ngIf="grid.rendered$ | async">
+    <button #btn igxButton="outlined" name="btnColumnHiding" (click)="toggle(btn, ref, actions)"
```

Comment:
@rkaraivanov Can you also add `title` to built-in buttons (at least icon ones) and include those in the localized resources? The native tooltip should be a nice improvement for usability and accessibility I suspect.